src/shared/lib/oql/oqlfilter.ts

```diff
@@ -1205,6 +1242,23 @@ const getGeneFromAlterations = (q: SingleGeneQuery) => {
     return (q.alterations as FUSIONCommandOrientationBase[])[0].gene;
 };
 
+export function fusionsInOQLQuery(oql_query: string): SingleGeneQuery[] {
```

Comment:
curious why you switched to "fusion" here when everything else in PR is structural variant

Reply:
Ah good point, I have renamed these methods to the SV equivalent.